dxr/plugins/python/indexers.py

```diff
@@ -21,6 +21,8 @@
             'py_function': QUALIFIED_NEEDLE,
             'py_derived': QUALIFIED_NEEDLE,
             'py_bases': QUALIFIED_NEEDLE,
+            'py_callers': QUALIFIED_NEEDLE,
+            'py_called_by': QUALIFIED_NEEDLE,
```

Comment:
I can't think of a very strong one. Maybe as part of some larger analysis rather than a user-facing filter. Should I remove it?